hermes-main-website/src/app/components/header/header.component.sass

```diff
@@ -0,0 +1,87 @@
+@import 'src/variables.sass'
+
+.content-menu li
+    display: inline-block
+    background-color: $red-main
+    font-size: 25px
+    margin:0
+    margin: 5px 10px
+
+.content-menu 
+    width: 100%
+    background-color: $red-main
+    display: flex
+    justify-content: flex-end
+    padding: 0.6rem 0 0.1rem
+    
+.content-menu li a:link, a:visited, a:hover 
+    text-decoration: none
+    color: $white-main
+    
+.header-logo 
+    height: 200px
+    width: 100%
+    background-color: $white-main
+    text-align: center
+    display: flex
+    flex-direction: column
+    align-items: center
+    position: absolute
+    margin-bottom: 20px
+    padding-bottom: 20px
+    
+.header-logo img
+    height: 80%
+    width: 100%vh
+
+#button-center
+    height: 20%
+    background-color: $red-main
+    font-size: 27px
+    color: $white-main
+    margin: 5px 10px
+    border-radius: 20px
+
+label
+    display: none
+
+#toggle 
+    display: none
+
+
+
+@media screen and ( max-width: 600px )
+    .content-menu 
+        background-color: $white-main
+        display: none
+
+    label
+        display: block
+        cursor: pointer
+
+    .content-menu
+        width: 100%
+        display: none
+
+    .content-menu li
+        display: block
+
+    #toggle:checked + .content-menu 
+        display: block
+
+    .content-menu li 
+        background-color: $white-main
+        text-align: left
+        margin: 0px 5px
```

Comment:
Combine margin and margin-top since they are the same attribute to have the same effect